bitrise.yml

```diff
@@ -84,12 +88,12 @@ workflows:
             envman add --key APPLICATION_VERSION_NAME --value "$APPLICATION_VERSION_NAME"
     - activate-ssh-key@4: {}
     - git-clone@8: {}
-    - path::./:
+    - path::./Users/vagrant/git:
```

Comment:
Had to change the path here as it seems that it works slightly differently on the Xcode stacks

<img width="1588" alt="image" src="https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/assets/25252536/8b5175a4-a7cd-4380-bb9f-71a506c1a912">

Here you can see that the directory we're put in at start has a different path than the usual `bitrise/src` for the Ubuntu stacks and that the path that the Xcode stack is looking for the `step.yml` is also not the same

Reply:
It looks like it's a bug from Bitrise: you are using a **relative path** (`./Users/vagrant/git`), but it's resolved as **an absolute path**.

Reply:
Here is the syntax for absolute and relative paths: https://devcenter.bitrise.io/en/references/steps-reference/step-reference-id-format.html#the-path---source

Note that they say relative paths are **relative to the folder containing `bitrise.yml`**.

Reply:
<img width="985" alt="image" src="https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/assets/9317502/15b924ce-e111-4207-beef-36387e0d6141">

Anyways, we lost too much time on this kind of buggy behavior.

Can you change `./Users/vagrant/git` to be `/Users/vagrant/git` everywhere? This will at least be correct.